pages/common/dotnet.md

```diff
@@ -0,0 +1,19 @@
+# dotnet
+
+> .NET Command Line Tools
+
+- Initialize a new .NET project
+
+`dotnet new project-type`
+
+- Restore nuget packages
+
+`dotnet restore`
+
+- Build and execute the .NET project in the current directory
+
+`dotnet run`
+
+- Run a packaged dotnet application (only needs the runtime, the rest require the .NET Core SDK installed)
```

Comment:
By "the rest" you mean a compiled CIL executable? Also, `require` should be `requires`.

Reply:
By rest I meant the rest of the commands.

There are two options for installing .NET Core:

1. The complete SDK (for development and build environments)
1. Only the runtime (for production environments)

The last command is the only command that the runtime supports. I used require because it refers to the _rest of the commands_ which is plural. Do you think we should get rid of the text in the parenthesis? Because what I've put there is not clear enough.

Reply:
Ahhhh cool! I see. With mono there's the `mono` command (the runtime), and `mcs` (the compiler).

And ahhhhh I see why that's not pluralised - I completely misinterpreted that lol! Perhaps `the rest of the commands require` would work better then.